styles/styles.css

```diff
@@ -16,6 +16,7 @@
   /* Colors */
   --bright-gray: #efefef;
   --gray: #979797;
+  --gray-box-shadow: #0003;
```

Comment:
@bstopp let me know if you want me to call this something else, or choose another color.

Reply:
I think almost all the box shadows are actually `rgba(0 0 0 / 20%)`? We should definitely look around and check.

Reply:
They are, I was just keeping in with hex codes.  I can easily go to the rgb format.